### PR TITLE
Adds topic selector

### PR DESCRIPTION
**Pull request by pakb**
[Test link](https://web-mapviewer.dev.bgdi.ch/feature_BGDIINF_SB-1665/index.html)

### CODE REVIEW

**Comment by pakb** [comment]
I don't get it, everything goes smoothly on local machine with `npm ci && npm run test:headless` but it fails on CI... Same Node version, same NPM version....

File: src/api/topics.api.js [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+        super(layerId, name, topicTypes.LAYER)
+        this.layerId = layerId
+    }
+}
+
+/**
+ * Reads the output of the topic tree endpoint, and creates all themes and layers object accordingly
+ *
+ * @param {TopicTreeItem} node The node for whom we are looking into
+ * @returns {TopicTreeItem}
```
Comment (ltshb) [inline src/api/topics.api.js:80]: `null` missing in the returns

File: src/api/topics.api.js [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+        return null
+    }
+}
+
+/**
+ * Loads the topic tree for a topic. This will be used to create the UI of the topic in the menu.
+ *
+ * @param {String} lang The lang in which to load the topic tree
+ * @param {Topic} topic The topic we want to load the topic tree
+ * @returns {TopicTreeItem[]} A list of topic tree nodes
```
Comment (ltshb) [inline src/api/topics.api.js:102]: I would add `@async` and correct the returns to `{Promise<Array<TopicTreeItem|null>>}`

**Review by ltshb — APPROVED** [review]
Just minor comments about jsdoc, but to be honest I don't understand much of the VUE stuff ;-)

**Pull request merged**